tasks/dependencies.rb

```diff
@@ -1,61 +1,169 @@
-class OverrideReader
-  def overrides
-    @overrides ||= {}
-  end
-  def override(name, **options)
-    overrides[name] = options
-  end
-end
+#
+# Copyright:: Copyright (c) 2016 Chef Software Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Once you decide that the list of outdated gems is OK, you can just
+# add gems to the output of bundle outdated here and we'll parse it to get the
+# list of outdated gems.
+#
+# We're starting with debt here, but don't want it to get worse.
+
+ACCEPTABLE_OUTDATED_GEMS = %w{
+  celluloid
+  celluloid-io
+  docker-api
+  fog-cloudatcost
+  fog-google
+  gherkin
+  google-api-client
+  jwt
+  mime-types
+  mini_portile2
+  retriable
+  rubocop
+  slop
+  timers
+  unicode-display_width
+  varia_model
+}
+
+require_relative "bundle_util"
+
+namespace :dependencies do
+  # Update all dependencies to the latest constraint-matching version
+  task :update => "dependencies:update_omnibus_overrides" do
+    extend BundleUtil
+    puts ""
+    puts "-------------------------------------------------------------------"
+    puts "Updating Gemfile.lock ..."
+    puts "-------------------------------------------------------------------"
+    bundle "update"
+
+    platforms.each do |platform|
+      puts ""
+      puts "-------------------------------------------------------------------"
+      puts "Updating Gemfile.#{platform}.lock ..."
+      puts "-------------------------------------------------------------------"
+      bundle "lock --update", gemfile: "Gemfile.windows", platform: platform
+    end
+
+    puts ""
+    puts "-------------------------------------------------------------------"
+    puts "Updating omnibus/Gemfile.lock ..."
+    puts "-------------------------------------------------------------------"
+    bundle "update", cwd: "omnibus"
+    # TODO make platform-specific locks for omnibus on windows, too
```

Comment:
Nope. Not entirely sure we care about this one as much, either. I didn't want to go *too* overboard with the windows locks. I'd like to pursue a solution in `bundler` itself.